src/index.html

```diff
@@ -9,6 +12,22 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <img class="img" src="images/logo.png"
+      alt="logo">
+      <nav class="nav">
+        <a class="link is-active" href="#apple">APPLE</a>
+        <a class="link" href="#samsung">SAMSUNG</a>
+        <a class="link" href="#smrtphones">SMARTPHONES</a>
+        <a class="link laptops" href="#laptops"
```

Comment:
follow naming convention block__element--modifier
in your case link should have class `nav__link--to-laptops-and-computers`